HousingSearchApi/V1/Infrastructure/Factories/ProcessesQueryGenerator.cs

```diff
@@ -36,9 +36,9 @@ public QueryContainer Create<TRequest>(TRequest request, QueryContainerDescripto
             _queryBuilder
                 .WithExactQuery(processListRequest.SearchText,
                     new List<string> { "patchAssignment.patchId" })
-                .WithExactQuery(processListRequest.TargetId?.ToString(),
+                .WithFilterQuery(processListRequest.TargetId?.ToString(),
```

Comment:
Any reason why these changed?

Reply:
Doesn't that mean that then the targetID request won't work?

Reply:
I guess it might not - but not sure how we can get both to work if you can't have more than one exact match

Reply:
I think you're overcomplicating by trying to make it into one query - just have an if statement on whether the request contains target id

If it contains target id: Do an exact match on target id & target type
If not: Do an exact match on search text